internal/git/git.go

```diff
@@ -118,12 +118,20 @@ func Verify(ctx context.Context, rekor rekor.Verifier, data, sig []byte) (*Verif
 	if err != nil {
 		return nil, fmt.Errorf("error getting signature certs: %w", err)
 	}
+	// TODO: This is very brittle, since the DER encoding will sort the
+	// certificates in the signature, which means that this ordering can
+	// change depending on the issuer.
+	// See https://en.wikipedia.org/wiki/X.690#DER_encoding
+	// We should find a better way to identify the leaf cert that was used
+	// to generate the signature.
+	cert := certs[len(certs)-1]
+
 	opts := x509.VerifyOptions{
 		Roots:     fulcioroots.Get(),
 		KeyUsages: []x509.ExtKeyUsage{x509.ExtKeyUsageCodeSigning},
```

Comment:
You will also need to add an Intermediates certpool to verify the certificate. You can add `Intermediates: fulcioroots.GetIntermediates()`

Reply:
Done!

(was actually not needed before since we were including the intermediate in the chain, but now that we're only including the leaf cert this was helpful 🙏 )